nusight2/src/client/components/button/button.tsx

```diff
@@ -24,10 +24,9 @@ export type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
 
 const ColorToClassName = {
   default:
-    "shadow-sm bg-white text-gray-600 ring-1 ring-inset ring-gray-300 enabled:hover:bg-gray-50 enabled:active:bg-gray-200 dark:bg-slate-800 dark:text-white dark:ring-slate-700 dark:enabled:hover:bg-slate-700 dark:enabled:hover:ring-slate-600 dark:enabled:active:bg-slate-800",
-  primary: "shadow-sm bg-nusight-500 text-white enabled:hover:bg-nusight-400 enabled:active:bg-nusight-600",
-  transparent:
-    "bg-transparent text-nusight-500 ring-transparent enabled:hover:bg-black/10 enabled:active:bg-black/20 dark:text-nusight-500 dark:enabled:hover:bg-white/10 dark:enabled:active:bg-black/20",
+    "shadow-sm bg-white text-gray-600 ring-1 ring-inset ring-gray-300 dark:bg-gray-600 dark:text-white dark:ring-gray-500",
```

Comment:
When using my colours, that approach looked better. I think it was also a little more flexible because the hover and click colours didn't need to be set explicitly. I've reverted to your surface colours and updated this (and similar changes) back to match them as per your recommendations.